typo: offine -> offline (#25795)
mdn/content: files/en-us/web/progressive_web_apps/guides/offline_and_background_operation/index.md

## Background operation

> While offline operation is probably the most common use for service workers, they also enable a PWA to operate even while the main app is closed. This is possible because the service worker is allowed to run while the main app is not running.

Reviewer: ```suggestion
While offline operations are the most common use for service workers, they also enable a PWA to operate even while the main app is closed. This is possible because the service worker can run while the main app is not running.
```
